docs/src/examples.md

```diff
@@ -70,39 +71,60 @@ crystal = Crystal(lat_vecs, [[0, 0, 0]], 227; setting="1")
+so we only need to specify the interaction on a single one of them. One nearest-neighbor bond is the one connecting basis index 1 with basis index 3 within a
 single unit cell. (We can figure this out using our tools for symmetry analysis, at the
-bottom of this page).
+bottom of this page). So, we can create a nearest-neighbor Heisenberg interaction just
+by specifying:
 
 ```julia
-J = 28.28           # Units of K
+J = 7.5413        # Units of K
 interactions = [
     heisenberg(J, Bond(1, 3, [0,0,0])),
 ]
 ```
 
 **(3)** Assembling a `SpinSystem` is straightforward -- provide the `Crystal`,
 the `Interaction`s, along with the dimensions of the simulation box in units of
-the lattice vectors. Then, we will randomize the system so that all spins are
-randomly placed on the unit sphere.
+the lattice vectors. We will also take this to be a spin-``3/2`` system with
+a identity ``g``-tensor, which we can specify by providing a `SiteInfo` on a single
+site (as they are all symmetry equivalent). We can also optionally omit this
+information, in which case all sites are assumed to be spin-1 with an identity
+``g``-tensor.
+
+Then, we will randomize the system so that all spins are randomly placed on the unit 
+sphere.
 
 ```julia
-sys = SpinSystem(crystal, interactions, (8, 8, 8))
+sites_info = [SiteInfo(1, 3/2, I(3))]
+sys = SpinSystem(crystal, interactions, (8, 8, 8), sites_info)
 rand!(sys)
 ```
 
 The `SpinSystem` type is the central type used throughout our simulations. Internally, it contains the spin degrees of freedom which evolve during the simulation as well as the Hamiltonian defining how this evolution should occur. This type can be indexed into as an array of size `B×Lx×Ly×Lz`, with the first index selecting the sublattice and the remaining three selecting the unit cell, and the spin located at the  will given. For example, `sys[2, 5, 5, 5]` gives the spin variable located on the second sublattice in the unit cell, which is fifth along each axis.
 
-**(4)** We will simulate this system using Langevin dynamics, so we need to create a [`LangevinSampler`](@ref). Note that the units of integration time and temperature are relative to the units implicitly used when setting up the interactions.
+Note that regardless of the spin magnitudes you specify on each site, indexing `sys`
+will always return a normalized unit vector.
+
+**(4)** We will simulate this system using Langevin dynamics, so we need to create a [`LangevinSampler`](@ref). Note that the units of integration time and temperature are relative to the units implicitly used when setting up the interactions. A rough
+rule of hand is that a reasonable value for integration timestep size is
+``Δt ≈ 0.02 / (S^2 * J)``.
 
 ```julia
-Δt = 0.02 / J       # Units of 1/K
-kT = 4.             # Units of K
+Δt = 0.02 / ((3/2)^2 * J) # Units of 1/K
+kT = 4.                   # Units of K
 α  = 0.1
-kB = 8.61733e-5     # Units of eV/K
+kB = 8.61733e-5           # Units of eV/K
```

Comment:
Yeah, the only place `kB` is used is down at the bottom of the example, in the plotting code so that I can plot frequency in terms of meV rather than K. So, I will move this line down to there.